FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -155,12 +156,33 @@ class IssuesViewController: UIViewController, UITableViewDelegate, UITableViewDa
                 DispatchQueue.main.async {
                     self.viewModel = viewModel
                     self.tableView.reloadData()
+                    self.moveForwardIfNeeded()
                 }
             }
         } else {
             tableView.reloadEmptyDataSet()
         }
         tableView.refreshControl?.endRefreshing()
+
+    }
+
+    func moveForwardIfNeeded() {
+        if let selectPath = UserDefaults.standard.string(forKey: UserDefaultsKey.selectIssuePath.rawValue) {
+            // set selected issue
+            let selectedIssue = issuesManager.issues.first(where: {$0.slug == selectPath})
```

Comment:
Curious if this is worth adding to `issuesManager` to make it easy to look up an issue from a slug from anywhere else? I can see that being useful.

Reply:
Added this to issuesManager instead